Documentation/SiteSets/Index.rst

```diff
@@ -18,7 +18,13 @@ our site package.
 
 In step :ref:`content-mapping-site-set` we added dependencies to our site set.
 
-..  contents::
+..  contents:: Table of Contents
```

Comment:
In that file the sentence "In step XY we XY created a basic site set XY" is very similar/duplicated

Reply:
Removed it